modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+import glob
+
+# Method used for evaluate corners coords 
+# Not used yet - need to re-write calculations with it
+def calc_coords(bb):
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+                   'cx': cx, 'cy': cy}
+    return coords
+
+# During calculation of intersection over union we are checking numerical value
+# of area_of_overlap, because if it is equal to 0 - we have no intersection
+def get_iou(
+    new_xmin, new_xmax, new_ymin, new_ymax, gt_xmin, gt_xmax, gt_ymin, gt_ymax):
```

Comment:
Function takes too many arguments. Better pass `new_coords` and `gt_coords`, and unpack them inside a function.

Reply:
I change it for all functions and visualization part.